config.example/inventory

```diff
@@ -58,5 +59,7 @@ slurm-node
 [all:vars]
 # SSH User
 #ansible_user=ubuntu
+#ansible_ssh_private_key_file='~/.ssh/id_rsa'
+# Configure SSH bastion/jumpbox for the cluster
```

Comment:
Sorry to nit, i think this comment line is from the old inventory missed in the merge. Otherwise looks good to me

Reply:
Ugh, you're right. There was some odd merge conflict. One sec.